### PR TITLE
Added dividends data extraction and display

### PR DESCRIPTION
**Pull request by MPalarya**

### CODE REVIEW

File: server/rest_api/routers/portfolios.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
+from collections import defaultdict
```
Comment (MPalarya) [inline server/rest_api/routers/portfolios.py:1]: remove

File: server/analysis/instrument.py [hunk 2]
```diff
@@ -11,7 +11,11 @@
 CLOSE_PRICE = "close_price"
 DATE = "transaction_date"
 UNITS = "units_accumulative"
+DIVIDENDS = "dividends_accumulative"
+VALUE = "value"
 CHANGE = "change_from_prev"
+CHANGE_WITH_DIVS = "change_with_divs_from_prev"
+USD_RATE = "usd_rate"
```
Comment (MPalarya) [inline server/analysis/instrument.py:18]: are they used?

File: client/src/components/charts/stackedBarChart.vue [hunk 3]
```diff
@@ -0,0 +1,110 @@
+<template>
```
Comment (MPalarya) [inline client/src/components/charts/stackedBarChart.vue:1]: WIP file

File: client/src/components/mainContent.vue [hunk 4]
```diff
@@ -105,6 +107,7 @@ export default {
       DataWasLoaded: false,
       barOptions: null,
       isDataFetched: false,
+      dividendsData: null,
```
Comment (dansterenson) [inline client/src/components/mainContent.vue:110]: Duplicated variable name "dividendsData"

File: client/src/components/charts/stackedBarChart.vue [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        const amount = this.date[j].value;
+
+        let prev_date = dates[date_counter];
+
+        if (curr_date !== prev_date) {
+          date_counter += 1
+          prev_date = dates[date_counter]
+        }
+
+        if (!symbol in result)
```
Comment (dansterenson) [inline client/src/components/charts/stackedBarChart.vue:94]: Did you mean "!(symbol in result)"

File: client/src/components/charts/stackedBarChart.vue [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+export default {
+  components: {
+    VueHighcharts,
+  },
+  props: ["data", "chartName"],
+
+  data() {
+    return {
+      barOptions: {
```
Comment (dansterenson) [inline client/src/components/charts/stackedBarChart.vue:18]: Add `credits: { enabled: flase,},` under "barOptions" in order to hide "highcharts.com"

File: client/src/components/mainContent.vue (hunk 4 above)
Comment (MPalarya) [reply client/src/components/mainContent.vue]: removed

File: client/src/components/charts/stackedBarChart.vue (hunk 5 above)
Comment (MPalarya) [reply client/src/components/charts/stackedBarChart.vue]: fixed

File: client/src/components/charts/stackedBarChart.vue (hunk 6 above)
Comment (MPalarya) [reply client/src/components/charts/stackedBarChart.vue:18]: done

**Review by BenSterenson — APPROVED** [review]

**Pull request merged**